javalib/src/main/scala/java/util/TimerTask.scala

```diff
@@ -0,0 +1,30 @@
+package java.util
+
+import scala.scalajs.js.timers
+
+abstract class TimerTask {
+  private[util] var cancelled: Boolean = false
+  private[util] var lastScheduled: Long = 0L
+
+  def run(): Unit
+
+  def cancel(): Unit = {
+    cancelled = true
+  }
+
+  def scheduledExecutionTime(): Long = lastScheduled
+
+  private[util] def doRun() {
+    if (!cancelled) {
+      lastScheduled = System.currentTimeMillis()
+      try {
+        run()
+      } catch {
+        case t: Throwable =>
+          cancelled = true
+          throw t
```

Comment:
Is that behavior on exception specified? In other words, should a repeated task actually be canceled if one of its runs throws an exception?